### DISCUSSION TITLE
HomeBrew 2.7.5 installation steps/commands

### DISCUSSION THREAD

**Discussion by aaskkrr** [Homebrew/discussions#695] [Getting started]
Hi,

I am trying to install homebrew latest(2.7.7) version in my MacBook. Installation was successful. On further proceeding to install maven/jenv or any formula, It fails in the git cloning of brew-core with the following message.

**fatal: the remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed**

I am thinking of uninstalling the current one and planning to install the 2.7.5 homebrew version. Is it possible? If so, Can you direct me with the right steps/commands please?

Thanks!

**Comment by SMillerDev** [comment]
That's not possible, homebrew will always use the latest brew version available. The issue you're seeing is in the connection between your device and github.com (that homebrew users to host the software) so downgrading won't even help.

**Reply by aaskkrr** [reply]
Thank you @SMillerDev. I have the speed of 200gbps and I have also tried installing during the (very :) )early hours of the day to avoid the traffic. It still fails.

Could it be possible to suggest other options available to install other than the one in the brew.sh homepage please?

Thanks!

**Comment by carlocab** [comment]
See https://github.com/Homebrew/discussions/discussions/666.

**Reply by aaskkrr** [reply]
Thank you @carlocab - I had gone through that post and I have similar issue while cloning the brew core..Cloning other general git repositories seem to be fine.
